.travis.yml

```diff
@@ -0,0 +1,2 @@
+language: java
+script: mvn -U -Pspark-2.0 test
```

Comment:
Already with default profile our CI is running , so it is better to run this travis CI with spark2.0 profile to verify.